### ISSUE TITLE
Graphs under Metrics tab must be superimposed/grouped by context subset by default

### ISSUE THREAD

**Issue by ashutoshsaboo** [aimhubio/aim#1533]
## 🚀 Feature

Currently say if i'm logging loss, accuracy, precision, recall, f1 etc for all train, validation, test sets. Then this results in 5*3=15 graphs on the metrics page under each run (these are way too many graphs and can be sometimes tough to analyze). I think this is not the ideal experience, and the ideal behaviour IMHO should be that these graphs must be by-default grouped based on context - for ex: for loss - there should be a single graph with 3 lines superimposed against the same axes - one for train, validation, test set respectively (along with a legend in the same graph showing which line corresponds to which subset). Similarly for accuracy, f1 etc. This is really what I look at when analyzing my models - I want to gauge if models are overfitting etc or how the performance is spanning across epochs for train/val/test set. These can only be easily inferred once they are superimposed on top of one another. 

I know the metric explorer already does this, but it's extremely inconvenient at this point to first navigate to metric explorer -> filter for specific run (you need to note down the run hash id correctly, which is again not user friendly) -> select specific metrics to group on -> and then visualize them. It's a cumbersome 4 step process which is quite time consuming and tough to do for each run. 

Ideal behaviour could be either to provide a 1-click explore button of some sort from the metrics page -> which "pre-fills" the query for that specific run and groups by context for all the metrics which that particular run is tracking (For a specific example: in this case for each of loss, f1, accuracy etc a separate graph grouped on context but only for that run must be filtered for in the metric explorer in the default setting). This would be the ideal solution in my view as the metric explorer already offers various toggles if someone wants a customized grouping functionality (from the default use-case). 

Another possibility could be to implement grouping by context and rendering of superimposed metric graphs under the metric tab of each run (similar to how the metric explorer does). But IMO here catch is from UX standpoint, there must be some toggle to not group or group graphs based on context (as some users might prefer for no grouping at all). 

### Additional context

cc: @SGevorg @gorarakelyan @rubenaprikyan @roubkar
